### ISSUE TITLE
Performance/Benchmark suite

### ISSUE THREAD

**Issue by jabley** [FusionAuth/java-http#2]
Would it be possible to publish the benchmark configurations? The netty numbers look odd, given Twitter and Apple do good traffic numbers with netty-based systems.

**Comment by robotdan** [comment]
Yes.. we need to do this. There is a bit of info in the `README.md`

> The load test configuration is set to 10 clients with 500,000 requests each. The client is Restify which is a FusionAuth library that uses `HttpURLConnection` under the hoods. All the servers were HTTP so that TLS would not introduce any additional latency.

But we need to publish a boiler plate harness  so that we can get more eyes on it to ensure we are measuring it objectively.
